lib/include/iot_taskpool.h

```diff
@@ -212,6 +217,7 @@ IotTaskPoolError_t IotTaskPool_SetMaxThreads( IotTaskPool_t * pTaskPool,
 /* @[declare_taskpool_createjob] */
 IotTaskPoolError_t IotTaskPool_CreateJob( const IotTaskPoolRoutine_t userCallback,
                                           void * const pUserContext,
+                                          IotTaskPoolJobStorage_t * const pJobStorage,
                                           IotTaskPoolJob_t * const pJob );
```

Comment:
If create job always uses a JobStorage, it probably doesn't need this parameter.

Reply:
It is needed for creating the job statically (i.e. _in place_).